otaclient/app/ota_client.py

```diff
@@ -219,31 +213,32 @@ def _download_file(entry: RegularInf) -> RegInfProcessedStats:
                 executor=_executor,
             )
             for _, task_result in _mapper.map(_download_file, download_list):
-                # task successfully finished
                 is_successful, entry, fut = task_result
+                # reset keep_failing timer on succeeded task or
+                # downloader is active during the last <POLL_INTERVAL> period.
+                if is_successful or self._downloader.is_downloader_active:
```

Comment:
I think `self._downloader.is_downloader_active` is not appropriate condition. Imagine if number of thread is 1.
keep_failing_timer should be reset if at least one byte can be donwloaded with the `download_file` function.

Reply:
Yes you are right! Let me reconsider a better condition.

Reply:
Updated in [698e01a](https://github.com/tier4/ota-client/pull/208/commits/698e01abae42ec4afe42b9d3395ba3fcbfed396b).